src/core/Browser.js

```diff
@@ -98,7 +98,7 @@
 		any3d: !window.L_DISABLE_3D && (ie3d || webkit3d || gecko3d) && !opera12 && !phantomjs,
 
 
-		// @property ie: Boolean
+		// @property mobile: Boolean
 		// `true` for all browsers running in a mobile devide.

```

Comment:
Could you fix this line to say "device" rather than "devide", while you're at it? :)